benchmark/sparse_blas/operations.cpp

```diff
@@ -705,8 +722,12 @@ const std::map<std::string,
          [](const Mtx* mtx) {
              return std::make_unique<SymbolicLuOperation>(mtx);
          }},
-        {"symbolic_cholesky", [](const Mtx* mtx) {
-             return std::make_unique<SymbolicCholeskyOperation>(mtx);
+        {"symbolic_cholesky",
+         [](const Mtx* mtx) {
+             return std::make_unique<SymbolicCholeskyOperation>(mtx, false);
+         }},
+        {"symbolic_cholesky_symmetric", [](const Mtx* mtx) {
+             return std::make_unique<SymbolicCholeskyOperation>(mtx, true);
```

Comment:
It is not great that the old name `symbolic_cholesky` does something different now compared to previously.
Would it not be possible to keep the old functionality with the old name and have a new name for the new functionality? Maybe `symbolic_cholesky_nonsymmetric`?
Since you were talking about reproducibility, this would make things significantly harder to reproduce an old benchmark that was using `symbolic_cholesky`.

Reply:
The benchmark has only been there for two days #1302, so I see no issue to renaming it

Reply:
Good point, I didn't know it was that young. In that case, I agree that there should not be any issue renaming it.